api/internal/repository/nft_token.go

```diff
@@ -75,6 +75,14 @@ func (n *nftTokenDbImpl) GetNfts(collectionAddresses []string, walletAddress str
 		query = query.Where("owner_address = ?", walletAddress)
 	}
 
+	if offset != nil {
+		query = query.Offset(*offset)
+	}
+
+	if limit != nil {
```

Comment:
there has to be an upper bound of the limit

Reply:
Yes. Added validation in the handler to make sure the limit is > 0 and <= 500